backend/api/login/views.py

```diff
@@ -62,7 +62,7 @@ def post(self, request, option):
         # TODO: fix error ( tree is not serializable )
         # request.session['tree'] = tree
         request.session.set_expiry(6000)
-        return JsonResponse({'success': True, 'message': str(request.session['list'])})
+        return JsonResponse({'success': True, 'message': request.session['list']})
```

Comment:
Oh, and I just realized that sending `request.session['list']` is for debugging.
You don't need to parse it if you don't want to.